tests/gold_tests/thread_config/check_threads.py

```diff
@@ -0,0 +1,83 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import psutil
+import argparse
+
+
+def count_threads(ts_path, etnet_threads, accept_threads):
+
+    for pid in psutil.pids():
```

Comment:
I suppose there is not a way to have autest pass the PID for the traffic server Process as an argument to this tool?

Reply:
After further discussion with @duke8253, I realized that the replay files need to be static (so that they can be replayed), and therefore we cannot encode a PID (which would be dynamic) into the arguments to the command.

So we need to search for the pid. Maybe someday the replay commands could store a pid file in the sandbox that could be read, but that change would be out-of-scope for this PR.